### DISCUSSION TITLE
Budgie application menu and menulibre don't reflect each other; how to fix?

### DISCUSSION THREAD

**Discussion by mclose90-cmd** [BuddiesOfBudgie/budgie-desktop#132] [Q&A] [answered]
I was editing my Budgie application menu with the 'Menu Editor' `menulibre` and while rearranging a couple of menu items with the up and down arrows it seems that my application menu and 'Menu Editor' no longer reflect each other. I'm not sure how it happened or if I've done something wrong. Specifically I was trying to move 'YaST' into the 'System Tools' menu, but now 'YaST' doesn't show up in 'Menu Editor' `menulibre`  at all, but 'YaST' does show up in the 'Applications' menu under 'All' and under the second 'System Tools' menu category. The menu categories for 'System Tools' and 'Other' have been doubled, but that hasn't been reflected in 'Menu Editor' `menulibre`. I've ran `update-desktop-database` as well with no positive affect. I've attached a video demonstrating what is going on with a HardInfo report. 

(I'm hoping someone could just point me to where I need to go to edit the menu entries manually myself.)

Gecko Linux 153 STATIC Budgie (bases upon openSUSE Leap 15.3 x86_64)
Kernel: 5.3.18-150300.59.68-default
Desktop Environment: Budgie
WM: Mutter (Budgie)

[HardInfo_Report_Gecko_Linux_153_STATIC_Budgie_openSUSE_15_3_Leap_html.tar.gz](https://github.com/BuddiesOfBudgie/budgie-desktop/files/8777787/HardInfo_Report_Gecko_Linux_153_STATIC_Budgie_openSUSE_15_3_Leap_html.tar.gz)

https://user-images.githubusercontent.com/88676245/170456793-10846937-d379-4258-970e-78561128f699.mp4

**Comment by EbonJaeger** [comment]
This sounds like something that will be fixed or addressed in the Budgie Menu rewrite (#110).

**Reply by mclose90-cmd** [reply]
That's great. Thank you. Any ideas of how and where to go to edit the menu manually to fix it?

**Comment by Solarunit** [comment]
I also noticed some problems with menulibre: editing an existing items just duplicates it and you can't remove one of them.
When using win+F2 search, one of the item is found; pinned on the Panel (modified) item is blinking too long when you click on it.

**Reply by mclose90-cmd** [reply]
> I also noticed some problems with menulibre: editing an existing items just duplicates it and you can't remove one of them. When using win+F2 search, one of the item is found; pinned on the Panel (modified) item is blinking too long when you click on it.

@Solarunit Try the `mozo` 'Menu Editor' from the MATE Desktop pattern. That worked for me, to undo what `menulibre` did. `mozo` can move and edit menu items for Budgie Desktop without messing things up.

**Comment by mclose90-cmd** [comment, marked as answer]
The `mozo` 'Menu Editor' from the MATE Desktop pattern can edit the application menu for Budgie Desktop. That worked for me, to undo what `menulibre` did. `mozo` 'Menu Editor' can move and edit menu items for Budgie Desktop without messing things up while also fixing any errors `menulibre` caused. 

Regards.
